packages/jest-types/src/Global.ts

```diff
@@ -119,3 +119,7 @@ export interface Global
     Omit<NodeJS.Global, keyof GlobalAdditions> {
   [extras: string]: unknown;
 }
+
+declare global {
+  const requireOutside: typeof require;
```

Comment:
I didn't really know where to put a global that's available to essentially all of our packages. Will putting it here make it available to our users? Who imports `@jest/types`?

Reply:
Oh you asked the same question. Yeah likely, but idk where to put it. A `.d.ts` file somewhere? A `@types/jest-internal-globals` package?

Reply:
might be best to just

```ts
const prettier: typeof import('prettier')
  // @ts-expect-error
  = requireOutside('prettier')
```

or something? just to avoid having to mess with globals